src/libraries/PID/DParticleID.cc

```diff
@@ -1221,6 +1237,23 @@ bool DParticleID::ProjectTo_SC(const DReferenceTrajectory* rt, unsigned int locS
 }
 
 // The routines below use the extrapolations vector from the track
+bool DParticleID::Distance_ToTrack(double locStartTime,const DTrackFitter::Extrapolation_t &extrapolation,const DFCALHit *locFCALHit,double &locDOCA,double &locHitTime) const{
+  if (fabs(locFCALHit->t-extrapolation.t-locStartTime)>OUT_OF_TIME_CUT)
```

Comment:
Is the default cut okay for these single hits pre-timewalk?

Reply:
The default cut is 35 ns -- I think this should be sufficient...